src/Features/Core/Portable/CodeRefactorings/AbstractRefactoringHelpersService.cs

```diff
@@ -336,13 +343,26 @@ protected virtual SyntaxNode DefaultNodeExtractor(SyntaxNode node, ISyntaxFactsS
                 return rightSide;
             }
 
+            // VB's arguments can have identifiers nested in ModifiedArgument -> we want
+            // indentifiers to represent parent node -> need to extract.
```

Comment:
you say this is about VB but you seem to eb doing it with both languages.  Either update comment, or update code.

Reply:
It can happen only in VB. The method always returns false in C# because there the identifier is always a Token (and thus the extraction happens automatically due to being first-level-Token child of wanted Node).

I know this isn't ideal solution but it was the best way to communicate the intent of the code best. I could create an override in VB version of the service but then the idea of this extraction is more burrier. I could also lift the `IsIdentifierOfParameter` to the service itself (because it might not be viewed as useful for both languages as it always returns false in C#).

Reply:
> I could also lift the IsIdentifierOfParameter to the service itself (because it might not be viewed as useful for both languages as it always returns false in C#).

this would be my preference.  Thanks! :)